.github/workflows/pr.yml

```diff
@@ -139,13 +139,27 @@ jobs:
     uses: ./.github/workflows/atoms-production-build.yml
     secrets: inherit
 
+  build-docs:
+    name: Production builds
```

Comment:
Left under `Production builds` so it lines up with all of the other builds we do